specs/asgi.rst

```diff
@@ -323,6 +329,7 @@ Version History
 ===============
 
 * 2.0 (2017-11-28): Initial non-channel-layer based ASGI spec
+* 2.1 (2018-07-01): Updated ASGI spec with a lifespan protocol.
```

Comment:
I'm not sure that a new protocol needs a new core ASGI version, to be honest. I do think adding the version key is sensible, though, but maybe just at a 2.0 for now.

Reply:
Changed.